openff/evaluator/__init__.py

```diff
@@ -3,11 +3,10 @@
 A physical property evaluation toolkit from the Open Forcefield Consortium.
 """
 
+from openff.units import unit
+
```

Comment:
This arguably should be dropped altogether, though it would break some amount of backwards compatibility.